tests/erc20/mod.rs

```diff
@@ -61,5 +61,13 @@ pub fn generate_cluster(
         }
     }
 
-    (state, txs)
+    let mut bytecodes = Bytecodes::new();
+    for account in state.values_mut() {
```

Comment:
This pattern is very unnatural. It is much better to declare `bytecodes` with `state` and `txs`, and make `ERC20Token` return `account` and `bytecode` separately. Or at least take the `code` from the returned account there instead of here.

Reply:
basically you want to cherry pick this commit? https://github.com/risechain/pevm/pull/236/commits/66da2bc94c6e98086a67468b241688ea0f99cd2e

Reply:
After looking at the Uniswap case this is much cleaner and less error-prone, and it's better to unify both instead of noisily change the `ERC20Token` interface. Let's just keep it as is.

Reply:
Removing `EvmAccount::code` is kinda nice since having `Some(code_hash)` and ` None` `code` is very confusing. Let's consider that in the next PR where we introduce the bytecodes map that accompany the accounts in the execution output. Changing the contract build interfaces can wait until then, too.